package.json

```diff
@@ -8,14 +8,43 @@
     "name": "app",
     "main": "src/app.js",
     "dependencies": {
-      "bootstrap": "npm:bootstrap@4.0.0-alpha.2",
+      "bootstrap": "github:twbs/bootstrap@4.0.0-alpha.2",
```

Comment:
I see, I also need to pull scss files how can achieve this ?    `"files": null,` ?